src/sessions/softphone-session-handler.ts

```diff
@@ -171,11 +171,11 @@ export class SoftphoneSessionHandler extends BaseSessionHandler {
     /* It really should only matter for connected calls as ended calls should have the logic already in place in SVH */
     if (this.isConnectedState(callState) && session) {
       if (callState.muted !== previousCallState?.muted) {
-        this.sdk.headset.setMute(callState.muted);
+        await HeadsetChangesQueue.queueHeadsetChanges(() => this.sdk.headset.setMute(callState.muted));
```

Comment:
So I don't think you need to await the headsetChangesQueue. See Example: https://stackblitz.com/edit/typescript-1yxa9i?file=index.ts

Was this due to linter complaints of dangling promises?

Reply:
I think this was more so just me taking precautionary measures to ensure the queue is in the order we expect.  But looking at your example, it does look unnecessary.  I'll remove it and test to see if it's still fine

Reply:
Okay yeah! It doesn't look like the await is needed